packages/core/src/Provider/TxSubmitProvider/types.ts

```diff
@@ -1,11 +1,12 @@
 // eslint-disable-next-line @typescript-eslint/ban-ts-comment
 // @ts-ignore
-import Cardano, { CardanoNodeErrors, Provider } from '../..';
+import Cardano, { CardanoNodeErrors, HandleResolution, Provider } from '../..';
 
 type SerializedTransaction = Cardano.util.HexBlob;
 
 export interface SubmitTxArgs {
   signedTransaction: SerializedTransaction;
+  context?: { handles?: HandleResolution[] | [] };
```

Comment:
It's already optional, so if we we're not setting the context we can do just `{signedTransaction}` or `{signedTransaction, context: {}}`
```suggestion
  context?: { handles?: HandleResolution[] };
```

Reply:
2d88f9ab41c0bf941b8126409e8d597fa2713303  should address this